### ISSUE TITLE
Isaac Lab missing OIGE envs

### ISSUE THREAD

**Issue by eferreirafilho** [NVIDIA-Omniverse/OmniIsaacGymEnvs#170]
Some OIGE envs such as Franka Deformable, Ingenuity and Factory ones, are not listed at

https://isaac-sim.github.io/IsaacLab/source/features/environments.html

Are there any plans on porting those?

Thanks!

**Comment by Mayankm96** [comment]
Yes, we are working on it. Still waiting to promise a timeline since we are a bit swamped with requests. Some of the environments will be made available soon.

If you're interested in porting the environments to Isaac Lab, we are happy to extend support for that.
